change type inventory (#37)
meaningfy-ws/rdf-differ: README.md

| Notation                   | Meaning                                                                                                                                                    | Example                 |
|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
| triple pattern < _s p o_ >   | each item in the triple represents a SPARQL variable or an URI. For brevity we omit the question mark prefix (?) otherwise the SPARQL reading shall apply. | i p v                   |
| arrow (_ --> _)              | state transition operator (from one version to the next)                                                                                                   | i1 p o  -->  i2 p o     |
| _i_ - in the triple pattern  | the instance subject (assuming class instantiation)                                                                                                        | i p v                   |

> | _p_ - in the triple pattern  | the main predicate                                                                                                                                         | i p v                   |

Reviewer: I recommend using the `code` text style for the **Notation** and **Example** columns as I think it should improve readability.
![table](https://i.imgur.com/2DVArSN.png)

| change type / pattern    | instance  | property value free  | property value language dependent | reified property value    | property value langauge dependent | reification object | Left condition checking | Right condition checking |
|---------------------------|-----------|----------------------|-----------------------------------|---------------------------|-----------------------------------|--------------------|-------------------------|--------------------------|
| Addition                  | 0  -->  i | 0  -->  i p v        | 0                                 | 0  -->  i p/op v          | 0                                 | 0                  |                       0 | x                        |
| Deletion                  | i  -->  0 | i p v  -->  0        | i p v@l  -->  0                   | i p/op v  -->  0          |                                   | 0                  | x                       |                        0 |

> | Value update              | 0         | i p v1  -->  i p v2  | i p v1@l  -->  i p v2@l           | i p/op v1  -->  i p/op v2 | i p/op v1@l  -->  i p/op v2@l     | 0                  | x                       | x                        |

Reviewer: same as previous recommendation

| Movement (cross instance) | 0         | i1 p v  -->  i2 p v  | 0                                 | i1 p/op v  -->  i2 p/op v | 0                                 | 0                  | x                       | x                        |
| Movement (cross property) | 0         | i p1 v  -->  i p2 v  | 0                                 | i p1/op v  -->  i p2/op v | 0                                 | 0                  | x                       | x                        |

> The state transition patterns presented in teh table above can be translated to SPARQL querries. The last two columns, referring to the quantification assumptions, are useful precisely for this purpose indicating what filters shall be used in the SPARQL query.  

Reviewer: ```suggestion
The state transition patterns presented in the table above can be translated to SPARQL queries. The last two columns, referring to the quantification assumptions, are useful precisely for this purpose indicating what filters shall be used in the SPARQL query.  
```

> Before we introduce the quantification assumptions, we need to mention that the current diffing is performed by subtracting teh new version of teh dataset from the old one resulting in the set of deletions between the two and, conversely, subtracting the old version of teh dataset from the new one resulting in a set of insertions between the two. Therefor we conceptualise four content graphs: _OldVersion_, _NewVersion_, _Insertions_ and _Deletions_. Below is teh table that summarises the quantification assumptions as conditions that apply to either left or right side of the transition operator and involve one of the four graphs introduced here.  

Reviewer: ```suggestion
Before we introduce the quantification assumptions, we need to mention that the current diffing is performed by subtracting teh new version of the dataset from the old one resulting in the set of deletions between the two and, conversely, subtracting the old version of the dataset from the new one resulting in a set of insertions between the two. Therefore we conceptualise four content graphs: _OldVersion_, _NewVersion_, _Insertions_ and _Deletions_. Below is the table that summarises the quantification assumptions as conditions that apply to either left or right side of the transition operator and involve one of the four graphs introduced here.  
```

> | Conditions on the left side of the transition operator                                                                                                                              | Consditions on the right side of the transition operator                                                                                                                            |

Reviewer: ```suggestion
| Conditions on the left side of the transition operator                                                                                                                              | Conditions on the right side of the transition operator                                                                                                                            |
```

|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | does NOT exists in the Insertion graph 				| exists in the Insertion graph 	|

Reviewer: ```suggestion
| does NOT exist in the Insertion graph 				| exists in the Insertion graph 	|
```

> | does NOT exists in the NewVersion graph [redundant]	| exists in the NewVersion graph [redundant]		|

Reviewer: ```suggestion
| does NOT exist in the NewVersion graph [redundant]	| exists in the NewVersion graph [redundant]		|
```

> | exist in the Deletions graph						| does NOT exist in the Deletions graph [redundant]		|

Reviewer: ```suggestion
| exists in the Deletions graph						| does NOT exist in the Deletions graph [redundant]		|
```

> | exist in the OldVersion graph [redundant] 			| does NOT exist in the OldVersion graph 	|

Reviewer: ```suggestion
| exists in the OldVersion graph [redundant] 			| does NOT exist in the OldVersion graph 	|
```
